[2023-03-29 16:28:42,973] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_22222222222.date_update_mrr 2023-03-29T16:28:37.827452+00:00 [queued]>
[2023-03-29 16:28:42,989] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_22222222222.date_update_mrr 2023-03-29T16:28:37.827452+00:00 [queued]>
[2023-03-29 16:28:42,990] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 16:28:42,992] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2023-03-29 16:28:42,993] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 16:28:43,019] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): date_update_mrr> on 2023-03-29T16:28:37.827452+00:00
[2023-03-29 16:28:43,028] {standard_task_runner.py:52} INFO - Started process 26971 to run task
[2023-03-29 16:28:43,036] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_22222222222', 'date_update_mrr', '2023-03-29T16:28:37.827452+00:00', '--job-id', '354', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpnp_vgefd', '--error-file', '/tmp/tmpyeily24g']
[2023-03-29 16:28:43,038] {standard_task_runner.py:77} INFO - Job 354: Subtask date_update_mrr
[2023-03-29 16:28:43,113] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_22222222222.date_update_mrr 2023-03-29T16:28:37.827452+00:00 [running]> on host 0bc3da97cebb
[2023-03-29 16:28:43,164] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_22222222222
AIRFLOW_CTX_TASK_ID=date_update_mrr
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T16:28:37.827452+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T16:28:37.827452+00:00
[2023-03-29 16:28:43,191] {python.py:151} INFO - Done. Returned value was: None
[2023-03-29 16:28:43,205] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=postgres_22222222222, task_id=date_update_mrr, execution_date=20230329T162837, start_date=20230329T162842, end_date=20230329T162843
[2023-03-29 16:28:43,259] {local_task_job.py:151} INFO - Task exited with return code 0
[2023-03-29 16:28:43,290] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
